hibernate-core/src/main/java/org/hibernate/sql/ast/spi/AbstractSqlAstTranslator.java

```diff
@@ -8178,6 +8178,10 @@ public void visitStandardTableDelete(TableDeleteStandard tableDelete) {
 						}
 					} );
 				}
+
+				if ( tableDelete.getWhereFragment() != null ) {
```

Comment:
Shouldn't you have something similar for update as well?

Reply:
Yeah, `AbstractSqlAstTranslator#visitStandardTableUpdate` already handled the additional where fragment the same way even before this PR.